test/helpers.ts

```diff
@@ -183,3 +184,12 @@ test('shouldFeatureRun', t => {
 		exclude: yesNo,
 	}), 'If any `exclude` is true, then it should not run, regardless of `asLongAs` and `include`');
 });
+
+test('getItemsBetween', t => {
+	t.deepEqual(getItemsBetween<number>([1, 10, 100], 10, 100), [10, 100]);
```

Comment:
Let's ensure that it works in the middle of the array + alt-clicking the same element

```suggestion
	t.deepEqual(getItemsBetween<number>([1, 10, 100, 1000], 10, 100), [10, 100]);
	t.deepEqual(getItemsBetween<number>([1, 10, 100, 1000], 10, 10), [10]);
```

Can you also replace 1/10/100/1000 with something more easily distinguishable? Colored emojis would be great. Let's make the initial array static too:

```js
const list = ['❤️', '💛', '💚', '💙']
```